source/includes/extracts-atlas-search-facts.yaml

```
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. If you frequently run case-insensitive regex queries 
+  (utilizing the ``i`` option), we recommend Atlas Search queries that 
```

I think we should avoid "we" because it can be a little unclear who is implied by "we". As in, is it "we as writers", "we the MongoDB company" ?

```suggestion
  (utilizing the ``i`` option), consider using Atlas Search queries that 
```